src/java/src/main/java/ai/onnxruntime/genai/Model.java

```diff
@@ -27,39 +27,6 @@ public Model(Config config) throws GenAIException {
     nativeHandle = createModelFromConfig(config.nativeHandle());
   }
 
-  /**
-   * Creates a Tokenizer instance for this model. The model contains the configuration information
-   * that determines the tokenizer to use.
-   *
-   * @return The Tokenizer instance.
-   * @throws GenAIException If the call to the GenAI native API fails.
-   */
-  public Tokenizer createTokenizer() throws GenAIException {
-    if (nativeHandle == 0) {
-      throw new IllegalStateException("Instance has been freed and is invalid");
-    }
-
-    return new Tokenizer(this);
-  }
-
-  // NOTE: Having model.createGeneratorParams is still under discussion.
```

Comment:
I'm guessing the discussion ended with the conclusion that neither `createTokenizer` or `createGeneratorParams` is needed.

Reply:
Do you want to keep `createTokenizer` and `createGeneratorParams`? I don't see these two in other language bindings. besides, I don't quite get the difference between `createGeneratorParams` and `new GeneratorParams(model)`.

Reply:
model.createGeneratorParams and new GeneratorParams(model) is arbitrary. You should do what makes the most sense in Java.

But, we want to remove 'createTokenizer' in the future, as you can just get the tokenizer from the model internally already. You can remove createTokenizer :)